backend/requirements.txt

```diff
@@ -2,52 +2,31 @@ aiohttp==3.9.5
 aiosignal==1.3.1
```

Comment:
What's the difference between `requirements-dev` and `requirements`?

Is there a test requirements?

Reply:
@palaniappan-r